tests/helm_template_generator.py

```diff
@@ -65,15 +66,16 @@ def validate_k8s_object(instance, kube_version="1.21.0"):
 
 
 def render_chart(
-    name="RELEASE-NAME",
-    values=None,
-    show_only=None,
-    chart_dir=None,
-    kube_version="1.21.0",
-    baseDomain="example.com",
+    name: str = "RELEASE-NAME",
+    values: dict = {},
```

Comment:
this and line 71 are bad. https://imgur.com/7UvO9VB
It's best to stick to None here, and there are other options like attr.s when you're initializing a class. Hit me up if you'd like more clarity

Reply:
this problem only happens with mutable objects, like dict and list. tuples are safe 
![image](https://user-images.githubusercontent.com/98349137/151036782-419886ca-d81f-40a9-a473-c29cf1f8d71c.png)

Reply:
great catch! i wonder if there's a linter for this common pitfall...